dt-posts/posts.php

```diff
@@ -310,19 +310,19 @@ public static function format_connection_message( $p2p_id, $activity, $action =
             }
         } else if ( $p2p_type === "groups_to_groups"){
             if ($action == "connected to"){
-                $object_note_to = sprintf( esc_html_x( '%s added to child groups', 'group2 added to child groups', 'disciple_tools' ), $from_title );
+                $object_note_to = sprintf( esc_html_x( '%s added to child groups', 'group1 added to child groups', 'disciple_tools' ), $from_title );
                 $object_note_from = sprintf( esc_html_x( '%s added to parent groups', 'group1 added to parent groups', 'disciple_tools' ), $to_title );
             } else {
-                $object_note_to = sprintf( esc_html_x( '%s removed from child groups', 'group2 removed from child groups', 'disciple_tools' ), $from_title );
+                $object_note_to = sprintf( esc_html_x( '%s removed from child groups', 'group1 removed from child groups', 'disciple_tools' ), $from_title );
```

Comment:
Change not needed.

Reply:
There were 2 strings the same but showed as different because the context was different - hence I changed this context to reflect the other stings' context.

Reply:
I only see one string about removing to child groups

Reply:
Reverted.